3.3-nginx/Dockerfile

```diff
@@ -5,32 +5,46 @@ LABEL maintainer="Chaim Sanders <chaim.sanders@gmail.com>"
 ARG COMMIT=v3.3/dev
 ARG BRANCH=v3.3/dev
 ARG REPO=SpiderLabs/owasp-modsecurity-crs
-ENV PARANOIA=1
-ENV ANOMALY_INBOUND=5
-ENV ANOMALY_OUTBOUND=4
+
+ENV PARANOIA=1 \
+    ANOMALY_INBOUND=5 \
+    ANOMALY_OUTBOUND=4 \
+    NGINX_KEEPALIVE_TIMEOUT=60 \
```

Comment:
Let's add the `s` suffix (for seconds). Also on 
From the nginx docs:

> It is recommended to always specify a suffix

https://nginx.org/en/docs/syntax.html